service/service.go

```diff
@@ -159,6 +176,8 @@ func New(config Config) (*Service, error) {
 // Boot starts top level service implementation.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
+		go s.operatorCollector.Boot(context.Background())
```

Comment:
The wiring here is outdated. The context should be passed in like here.

https://github.com/giantswarm/cluster-operator/blob/0ed29d8467a3cd7d1c2c46406dcd2dbd0b6b0542/service/service.go#L283